### PR TITLE
minor docstring and typehint improvements

### PR DESCRIPTION
**Pull request by bendichter**

### CODE REVIEW

File: src/neuroconv/datainterfaces/ecephys/baserecordingextractorinterface.py [hunk 1]
```diff
@@ -185,7 +186,7 @@ def run_conversion(
         else:
             recording = self.recording_extractor
 
-        write_recording(
+        return write_recording(
```
Comment (bendichter) [inline src/neuroconv/datainterfaces/ecephys/baserecordingextractorinterface.py:189]:
```suggestion
        write_recording(
```

File: src/neuroconv/tools/spikeinterface/spikeinterface.py [hunk 2]
```diff
@@ -908,10 +909,12 @@ def add_all_to_nwbfile(
         Dictionary of options for the RecordingExtractorDataChunkIterator (iterator_type='v2')
         or DataChunkIterator (iterator_type='v1').
         Valid options are
-            buffer_gb : float (optional, defaults to 1 GB, available for both 'v2' and 'v1')
-                Recommended to be as much free RAM as available). Automatically calculates suitable buffer shape.
-            chunk_mb : float (optional, defaults to 1 MB, only available for 'v2')
-                Should be below 1 MB. Automatically calculates suitable chunk shape.
+            buffer_gb : float, defaults: 1.0
```
Comment (bendichter) [inline src/neuroconv/tools/spikeinterface/spikeinterface.py:912]:
```suggestion
            buffer_gb : float, default: 1.0
```

File: src/neuroconv/tools/spikeinterface/spikeinterface.py [hunk 3]
```diff
@@ -1362,18 +1364,21 @@ def add_units_table(
...
 ):
     """
     Auxiliary method for adding waveforms to an existing units_table.
 
     Parameters
     ----------
-    sorting :  A spikeextractors SortingExtractor.
-    units_table : a previously created units table
-    skip_features : list of str
+    sorting : SortExtractor
```
Comment (CodyCBakerPhD) [inline src/neuroconv/tools/spikeinterface/spikeinterface.py:1376]:
```suggestion
    sorting : SortingExtractor
```

Comment (CodyCBakerPhD) [reply src/neuroconv/tools/spikeinterface/spikeinterface.py:1376]: To match the explicit parent module used in other places, such as `pynwb.NWBFile`

**Comment by CodyCBakerPhD** [comment]
One typo/Improvement otherwise LGTM

File: src/neuroconv/tools/spikeinterface/spikeinterface.py [hunk 4]
```diff
@@ -20,7 +20,7 @@
 
 from .spikeinterfacerecordingdatachunkiterator import SpikeInterfaceRecordingDataChunkIterator
 from ..nwb_helpers import get_module, make_or_load_nwbfile
-from ...utils import dict_deep_update, FilePathType, calculate_regular_series_rate
+from ...utils import dict_deep_update, calculate_regular_series_rate, FilePathType
```
Comment (bendichter) [inline src/neuroconv/tools/spikeinterface/spikeinterface.py:23]:
```suggestion
from ...utils import dict_deep_update, FilePathType, calculate_regular_series_rate
```

Comment (CodyCBakerPhD) [reply src/neuroconv/tools/spikeinterface/spikeinterface.py]: @bendichter if you like, we can setup `isort` in pre-commit and let it do the work for this kind of stuff

**Review by CodyCBakerPhD — APPROVED** [review]

**Comment by CodyCBakerPhD** [comment]
Auto-merge is pretty cool

File: src/neuroconv/tools/spikeinterface/spikeinterface.py (hunk 4 above)
Comment (bendichter) [reply src/neuroconv/tools/spikeinterface/spikeinterface.py]: > @bendichter if you like, we can setup `isort` in pre-commit and let it do the work for this kind of stuff

Yeah, that seems like a good idea

**Pull request merged**